website/custom.css

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+:root {
+  --ifm-color-primary: #25c2a0 !important;
```

Comment:
Why still need `!important`?

Reply:
idk why but it got overridden by infima default css. must be some css ordering problem. Important also make it clears to be an override ?